### PR TITLE
Feature/cvsb 3952

### PR DESCRIPTION
**Pull request by oalexandru**

### CODE REVIEW

File: src/main/java/uk/gov/dvsa/model/cvs/certificateData/AdrPassCertificateData.java [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+    public String getFormattedPermittedDangerousGoods() {
+        String formattedPermittedDangerousGoods = "";
+        for (String permittedDangerousGood : this.permittedDangerousGoods) {
+            formattedPermittedDangerousGoods = permittedDangerousGood + "     ";
+        }
+        return formattedPermittedDangerousGoods;
+    }
+
+    public boolean getFormattedSubstancesPermitted() { // returns true for the first value that tankStatement can have and false for the other one so it can be processed in view
+        if(this.tankStatement.getStatement().equals("Substances permitted under the tank code and any special provisions specified in 9 may be carried")){
```
Comment (KHodderDVSA) [inline src/main/java/uk/gov/dvsa/model/cvs/certificateData/AdrPassCertificateData.java]: Enum?

File: src/main/java/uk/gov/dvsa/model/cvs/certificateData/AdrPassCertificateData.java [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public boolean getIsExplosivesType2() {
+        for (String permittedDangerousGood : this.permittedDangerousGoods) {
+            if(permittedDangerousGood.equals("Explosives (type 2)")){
```
Comment (KHodderDVSA) [inline src/main/java/uk/gov/dvsa/model/cvs/certificateData/AdrPassCertificateData.java]: More Enums?

File: src/main/java/uk/gov/dvsa/model/mot/enums/DocumentsConfig.java [hunk 3]
```diff
@@ -79,6 +82,7 @@
     }
 
     public static DocumentsConfig fromDocumentName(String name) {
+        System.out.println("CHECK DOCUMENT NAME HERE" + name);
```
Comment (KHodderDVSA) [inline src/main/java/uk/gov/dvsa/model/mot/enums/DocumentsConfig.java]: For testing? Or redundant?

File: src/main/resources/views/CommercialVehicles/ADR_PASS.hbs [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+<div class="permittedDangerousGoods">
+    <h3>7 Vehicle designation(s) according to 9.1.1.2 of ADR</h3>
+    <p><span id="formattedPermittedDangerousGoods">{{adrData.formattedPermittedDangerousGoods}}</span></p>
+</div>
+<div class = "brakeEndurance">
+    <h3>8 Endurance braking system</h3>
+    {{#if adrData.brakeEndurance}}
+        <p>( )Not applicable (<img src="{{root}}/assets/images/tick.png" alt="tick" class="tick" id="tick"/>)The effectiveness according to 9.2.3.1.2 of ADR is sufficient for a total mass of transport unit of (<span id="weight">{{adrData.weight}}</span>)t</p>
+    {{else}}
+        <p>(<img src="{{root}}/assets/images/tick.png" alt="tick" class="tick" id="tick"/>)Not applicable ( )The effectiveness according to 9.2.3.1.2 of ADR is sufficient for a total mass of transport unit of (XX)t</p>
```
Comment (KHodderDVSA) [inline src/main/resources/views/CommercialVehicles/ADR_PASS.hbs]: Is XX supposed to be a value?

File: src/main/java/uk/gov/dvsa/model/cvs/certificateData/AdrPassCertificateData.java (hunk 2 above)
Comment (oalexandru) [reply src/main/java/uk/gov/dvsa/model/cvs/certificateData/AdrPassCertificateData.java]: done

File: src/main/java/uk/gov/dvsa/model/cvs/certificateData/AdrPassCertificateData.java (hunk 1 above)
Comment (oalexandru) [reply src/main/java/uk/gov/dvsa/model/cvs/certificateData/AdrPassCertificateData.java]: done

File: src/main/java/uk/gov/dvsa/model/mot/enums/DocumentsConfig.java (hunk 3 above)
Comment (oalexandru) [reply src/main/java/uk/gov/dvsa/model/mot/enums/DocumentsConfig.java]: redundant. took it out

File: src/main/resources/views/CommercialVehicles/ADR_PASS.hbs (hunk 4 above)
Comment (oalexandru) [reply src/main/resources/views/CommercialVehicles/ADR_PASS.hbs]: no. xx is a value only when the 2nd option is ticked in the view(when if condition is true). when first option is ticked in the view(else branch) xx doesn't get populated.

**Pull request merged**